tools.yaml

```diff
@@ -3,7 +3,7 @@
 tools:
 - github.com/go-swagger/go-swagger/cmd/swagger@v0.29.0
 - github.com/gordonklaus/ineffassign@v0.0.0-20210914165742-4cc7213b9bc8
-- github.com/goreleaser/goreleaser@v1.18.2
+- github.com/goreleaser/goreleaser@v1.19.3-0.20230717115619-dbcebd00ae61
```

Comment:
I think we should probably wait for an official release for this.  Looks to be on 1.19.2 at the moment.

Reply:
yes, for now it is the latest. 1.19.3 should work fine.
I will upgrade to the latest whenever it will available.

Reply:
I don't really understand where this package is coming from if its not official released yet.  So I think we should wait until 1.19.3 is official and then just use the official package.

Reply:
To install from the main branch yesterday i did `go install github.com/goreleaser/goreleaser@main`
And than i checked the version of the goreleaser --version it was  to the last commit `v1.19.3-0.20230717115619-dbcebd00ae61`
so i added that version rather than pointing to the main branch because main can have bug, but upto that all was working fine for our use cases